images/linux/scripts/installers/oc.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+################################################################################
+##  File:  aws.sh
+##  Desc:  Installs the AWS CLI
+################################################################################
+
+# Source the helpers
+source $HELPER_SCRIPTS/document.sh
+# Install the oc CLI
+curl "https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.6/linux/" > oc.tar.gz
```

Comment:
Is it possible to get the latest version instead of hardcoding the exact one?

Reply:
yes sorry I changed this last minute :X

Reply:
changed to latest